easy/7.js

```diff
@@ -1,5 +1,7 @@
 function missingAngle(angle1, angle2) {
-	
+    if(angle1+angle2<90) return "obtuse";
+    else if(angle1+angle2==90) return "right";
+    else return "acute";

```

Comment:
The last else is optional. If previous conditions weren't fulfilled it return 'acute'

Reply:
yes, i know

Reply:
so i must remove it? or it can be there?